calcom/cal.com
# feat: posthog app added

This PR is to add Posthog analytics in the app store

fixes: #13546 

loom video:
https://www.loom.com/share/a7e6dc1389c54886b9bc627440d463bb

## Type of change



- New feature (non-breaking change which adds functionality)


## How should this be tested?

- Add Posthog analytics application to an event type.
- Open up the event page and it should automatically track it.

## Mandatory Tasks
